packages/rich-list-core/src/controller/AddressParser/dftx/accountToUtxos.ts

```diff
@@ -0,0 +1,12 @@
+import { fromScript } from '@defichain/jellyfish-address'
+import { AccountToUtxos, CAccountToUtxos, DfTx } from '@defichain/jellyfish-transaction'
+import { DfTxAddressParser } from './_abstract'
+
+export class AccountToUtxosParser extends DfTxAddressParser<AccountToUtxos> {
```

Comment:
Regarding the naming for this file, `accountToUtxos.ts` is used. But the test file for this is named as `AccountToUtxos.test.ts`. Should we change the name for this to `AccountToUtxos.ts` instead?